jobclient/python/setup.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+
+from setuptools import setup
+
+from cook.scheduler.client import CLIENT_VERSION
+
+with open('README.md') as fd:
+    readme = fd.read()
+
+requirements = [
+    'requests'
+]
+
+setup(name='cook-scheduler-client',
```

Comment:
I like the fact that we didn't call this "jobclient" because it gives us flexibility to include things other than just job management in this API. We also avoid people calling the library "the job client", which might be confused with other things that manage "jobs" in other contexts. I think "cook (scheduler) client" is both clear and concise. 👍